task14/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>JS課題14</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <div id="js-parent"></div>
+    <div id="modalPlace"></div>
```

Comment:
IMO
As with other ID names, when used in scripts, it is better to add It is better to add `js-`.

Reply:
You're right!
Thanks for pointing that out.